### DISCUSSION TITLE
[Project beta] Project readme

### DISCUSSION THREAD

**Discussion by max397574** [community/community#11445] [Projects and Issues] [answered]
There is a project readme which you can write in the settings.
Where can users see this?
I think it should be there when you click on the project for the first time.
It could just be a separate tab like the views are.

**Comment by hiroshi-dewa** [comment, marked as answer]
右上のボタンをクリックすると、プロジェクトのREADMEが右側にモーダル表示されますが、READMEが縦に長いと表示しきれず、スクロールもできません。
改善をお願いします。
また、READMEのモーダルの横幅をIssueのものと同じぐらいにしてほしいです。

![image](https://user-images.githubusercontent.com/20852143/165274759-967cade0-16d5-47fa-a2c1-78ab58072fe0.png)
